pkg/apiserver/event/event.go

```diff
@@ -56,18 +58,19 @@ func Register(r *gin.RouterGroup, s *Service) {
 
 	// TODO: add more api handlers
 	endpoint.GET("/", s.listEvents)
```

Comment:
This API should be `endpoint.GET("", s.listEvents)`. Would you mind updating this in this PR?

```
curl localhost:2333/api/events
```